src/core/protocol-authorization.ts

```diff
@@ -274,6 +275,21 @@ export class ProtocolAuthorization {
 
     const actionRules = inboundMessageRuleSet.$actions;
     if (author === tenant) {
+      // tenant is always authorized
+      return;
+    } else if (incomingMessage.author !== undefined && incomingMessage.authorizationPayload?.permissionsGrantId !== undefined) {
+      // PermissionsGrant gives the author explicit access to this record
```

Comment:
Sanity: sorry for the monkey-wrench, I noticed `verifyActionCondition()` call in line 73 will seem to cause problem if the initial content is written not by the author of the incoming message.

I am fine if we file an issue just to get this PR merged and address this in an immediate follow up PR.

Reply:
Let's address this in a follow up PR. The tenant has this same restriction. We should all get on the same page the desired behavior for who can update a protocol record they did not initially author.
https://github.com/TBD54566975/dwn-sdk-js/issues/476